FPGA22_HIER_SKY_PNR/FPGA22_HIER_SKY_Verilog/OpenFPGAEngine.info

```diff
@@ -0,0 +1,55 @@
+commit 48b2bff0d909e2c6d0740d2a5386123eb238349f
+Author: tangxifan <tangxifan@gmail.com>
```

Comment:
No sure if this file should be uploaded

Reply:
This is to log which commit version has been used for netlist generation

Reply:
O.K. Let's keep it for now. But users should use the master of OpenFPGA all the times. We ensure backward compatibility.